authority/provisioner/dpop_options.go

```diff
@@ -0,0 +1,55 @@
+package provisioner
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"text/template"
+)
+
+type DPOPOptions struct {
+	// ValidationExecPath is the name of the executable to call for DPOP
+	// validation.
+	ValidationExecPath string `json:"validation-exec-path,omitempty"`
+	// Backend signing key for DPoP access token
+	SigningKey string `json:"key"`
+	// URI template acme client must call to fetch the DPoP challenge proof (an access token from wire-server)
+	DpopTarget string `json:"dpop-target"`
```

Comment:
Here and before, we usually use cammelCase for JSON properties.

Reply:
Will update this in follow up. Didn't do so to not break the e2e test.

Reply:
Changed this in https://github.com/smallstep/certificates/pull/1671